### DISCUSSION TITLE
is v7.8.0 legitimate?

### DISCUSSION THREAD

**Discussion by nypinstripes** [TanStack/table#3929] [General] [answered]
Hello, 

Last couple of days when i run yarn outdated, i see there is a new package version for react-table 7.8.0. However when I look on the releases page I see it jumps from 7.7.0 straight to 8.0.0-alpha-1. I'm just a little concerned about installing, because I don't know what is in it or that it might have been illegitimate.

@tannerlinsley 
@KevinVandy

**Comment by tannerlinsley** [comment, marked as answer]
Good catch. Here is the release: https://github.com/TanStack/react-table/releases/tag/v7.8.0
